falcon_kit/bash.py

```diff
@@ -294,6 +294,8 @@ def script_run_consensus(config, db_fn, las_fn, out_file_bfn):
     params.update(locals())
     if config["falcon_sense_skip_contained"]:
         run_consensus = """LA4Falcon -H$CUTOFF -fso {db_fn} {las_fn} | """
+    elif config["falcon_greedy"]:
```

Comment:
Ooops. I approved too quickly. `falcon_kit/bash.py` needs to be changed too.

There are three kinds of option names:

1. pbsmrtpipe
2. `fc_run.cfg` (or whatever the user names this)
3. Internal dictionary, based on `fc_run.cfg`

I like to let 2 match 3, though not strictly necessary. 1 can be completely different.

This PR now has 2 match 3, but `bash.py` assumes `falcon_greedy` in the internal dictionary.